SimPEG/EM/NSEM/SurveyNSEM.py

```diff
@@ -64,27 +83,31 @@ def eval(self, f):
     def evalDeriv(self, f):
         raise Exception('Use Sources to project fields deriv.')
 
-#################
+#########
 # Data
-#################
+#########
 
 
-class Data(SimPEGsurvey.Data):
-    '''
+class Data(SimPEGsurvey.Data, DataNSEM_plot_functions):
```

Comment:
I am not sure of thoughts on this, but I wonder if we should not overwrite the main `Data` name but rather call this `DataNSEM` or something? That way there isn't any naming clashes?